src/frontend/src/views/Data/cellRenderers.tsx

```diff
@@ -179,7 +178,7 @@ const TREE_CUSTOM_RENDERERS: Record<string | number, CellRenderer> = {
     <TreeTypeTooltip value={value as string}>
       <TreeRowContent>
         <UnderlinedCell data-test-id={`row-${header.key}`}>
-          {value}
+          {`${value}`}
```

Comment:
```suggestion
          {value}
```

Reply:
All of these are related to typescript.  I'm definitely open to suggestions if there's a more elegant way to address the type errors.

![Screen Shot 2022-08-22 at 9 53 56 AM](https://user-images.githubusercontent.com/109251328/185976659-3035034a-cb1c-452d-a3c7-cade79613a98.png)

Reply:
I only read through the various comments on the PR, not the actual files changed, so I might be missing additional context, but here's my thoughts on this:

- If the only reason it's here is to do a type cast in to a string, I think it's clearer to the reader to do `String(value)` than the string interpolation, even if the result is the same.
- I think the root cause here is how generic the parent components and data being moved around is trying to be. From my experience messing with table stuff in the past, a lot of this comes down to everything being written in a very generic way at a high level, but then the actual low level display of data not being very generic. But instead of an explicit narrowing moment somewhere in the middle, it's more kinda hand-waved.
  - I'm curious how this was passing without error before? Was there a type assertion or something that got removed?
- If I'm right and the root cause is that types are just rather amorphous in regards to our table implementation, I don't think there's any value in figuring out a good solution right now. When Maya refactors the tables, a lot of this will have to change anyway, and we can either fix it then or the refactoring itself will do the fixing.